addons/website_livechat/static/src/core_ui/thread_patch.xml

```diff
@@ -5,7 +5,7 @@
             <t t-set="visitor" t-value="props.thread.visitor"/>
             <div t-if="visitor and !env.inChatWindow" class="o-website_livechat-VisitorBanner py-4 px-2 d-flex border-bottom">
                 <div class="o-website_livechat-VisitorBanner-sidebar me-2 d-flex justify-content-center">
-                    <img class="rounded o-website_livechat-VisitorBanner-avatar" t-att-src="threadService.avatarUrl(visitor, props.thread)" alt="Avatar"/>
+                    <img class="rounded o-website_livechat-VisitorBanner-avatar" t-att-src="threadService.avatarUrl(props.thread.correspondent, props.thread)" alt="Avatar"/>
```

Comment:
Probably in follow-up PRs to remove `thread.visitor` and rely on `thread.correspondent` all the time